### PR TITLE
new yaml file

### PR DESCRIPTION
**Pull request by bvennam**

### CODE REVIEW

**Comment by bvennam** [comment]
@duglin review, please.

File: fib-service.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: fibonacci-app
```
Comment (duglin) [inline fib-service.yaml]: did you mean to change the name?

Comment (bvennam) [reply fib-service.yaml]: yes, in the case that they don't delete it, i don't want these to conflict.

Comment (duglin) [reply fib-service.yaml]: oh, I thought we'd just upgrade the same app to include the builder.  Then they're slowly growing the app's infrastructure as they grow their knative expertise. Similar to how we keep the same app when we switch to knctl.

Comment (bvennam) [reply fib-service.yaml]: okay - updated.

**Comment by bvennam** [comment]
@duglin this one is ready for review

**Review by duglin — APPROVED** [review]

**Pull request merged**